obs/window-basic-main.cpp

```diff
@@ -80,6 +80,11 @@ OBSBasic::OBSBasic(QWidget *parent)
 {
 	ui->setupUi(this);
 
+	QString styleSheetPath(os_get_config_path("obs-studio/stylesheet.qss"));
+	QFile styleSheet(styleSheetPath);
+	styleSheet.open(QFile::ReadOnly);
+	this->setStyleSheet(styleSheet.readAll());
```

Comment:
Doing 'this->' is sort of redundant unless you're explicitly avoiding a name conflict.